client/layout/nps-survey-notice/index.jsx

```diff
@@ -58,6 +59,9 @@ class NpsSurveyNotice extends Component {
 			// (2) the user notices the notice more, since it will cause a change to the
 			//     screen they are already looking at
 			setTimeout( this.props.showNpsSurveyNotice, 3000 );
+
+			analytics.mc.bumpStat( 'calypso_nps_survey', 'displayed' );
```

Comment:
Change to 'notice_displayed'.

We could also add a separate stat bump for when the actual survey was displayed -- that would let us look at things in a single chart in MC.